pydeconz/models/sensor/moisture.py

```diff
@@ -29,3 +36,16 @@ def moisture(self) -> int:
         0-100 in percent.
         """
         return self.raw["state"]["moisture"]
+
+    @property
+    def scaled_moisture(self) -> float:
```

Comment:
I think this can be skipped. As mentioned with the other PR it's better to leave the scaling to the end user

Reply:
I understand that with rounding, but also scaling?

Why are humidity and temperature still use scaling then?
Leaving this out, would mean, that it is handled different between humidity and moisture?

Reply:
Sorry. Yes indeed. Doing this quickly on a phone was a bad idea.